dashboard/assets/packages/report/src/components/Chart/Chart.utils.ts

```diff
@@ -0,0 +1,73 @@
+import uPlot, { type Axis, type Options, type Series } from "uplot"
+import { type UnitType } from "@xk6-dashboard/model"
+import { format, tooltipPlugin, type SeriesPlot } from "@xk6-dashboard/view"
+
+import { common, grey } from "theme/colors.css"
+
+import * as styles from "./Chart.css"
+
+const AXIS_SIDE = 1
+const AXIS_SIZE = 70
+
+const sync = uPlot.sync("chart")
+
+const dateFormats = [
+  [3600 * 24 * 365, null, null, null, null, null, null, null, 1],
+  [3600 * 24 * 28, null, null, null, null, null, null, null, 1],
+  [3600 * 24, null, null, null, null, null, null, null, 1],
+  [3600, "{HH}", null, null, null, null, null, null, 1],
+  [60, "{HH}:{mm}", null, null, null, null, null, null, 1],
+  [1, ":{ss}", null, null, null, null, null, null, 1],
+  [0.001, ":{ss}.{fff}", null, null, null, null, null, null, 1]
+]
```

Comment:
@szkiba I've messed around with these formats to remove the date, could you take a look at this and confirm whether it's okay?